kafka/channel/pkg/utils/util.go

```diff
@@ -89,3 +91,13 @@ func TopicName(separator, namespace, name string) string {
 	topic := []string{knativeKafkaTopicPrefix, namespace, name}
 	return strings.Join(topic, separator)
 }
+
+func FindContainer(d *appsv1.Deployment, containerName string) *corev1.Container {
```

Comment:
I couldn't find a function like this in the codebase. Any pointers?

Reply:
hrm...

Reply:
same. I ended-up implemented it. https://github.com/knative/eventing/blob/master/pkg/reconciler/pingsource/pingsource.go#L198

Reply:
/hold
Ok, looks like I need to write some unit tests...

Reply:
/unhold

Done with unit tests.
Next time I need this, I will refactor it into a function in knative/pkg